atom/renderer/api/atom_api_web_frame.cc

```diff
@@ -344,6 +344,34 @@ void WebFrame::SetIsolatedWorldHumanReadableName(int world_id,
       world_id, blink::WebString::FromUTF8(name));
 }
 
+void WebFrame::SetIsolatedWorldInfo(int world_id, mate::Arguments* args) {
+  if (args->Length() < 1) {
+    args->ThrowError("Invalid args");
+    return;
+  }
+
+  mate::Dictionary options;
+  if (!args->GetNext(&options)) {
```

Comment:
Why do we use `args` to fetch this if it is a required parameter.  See above suggestion for change to function signature